pipelines/release-to-mrrc/release-to-mrrc.yaml

```diff
@@ -0,0 +1,204 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: release-to-mrrc
+  labels:
+    app.kubernetes.io/version: "4.1.0"
```

Comment:
We typically start with 0.1.0. Any reason to start with 4.1.0?

Reply:
This is a copied value from other release pipeline. Changed to 0.1.0 in new commit.